app/src/pages/TokenPage/TokenPage.tsx

```diff
@@ -123,6 +138,66 @@ export const TokenPage = (): React.ReactElement => {
+                  <Stack direction={Direction.Horizontal} isFullWidth={true} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
+                    <Box variant='tokenTable' width='12rem' height='3rem'>
+                      <Account accountId={tokenSale?.fromAddress} />
+                    </Box>
+                    <Stack.Item alignment={Alignment.Center}>
+                      <Box variant='tokenTable' width='6rem' height='3rem'>
+                        <KibaIcon iconId='ion-arrow-forward' />
+                      </Box>
+                    </Stack.Item>
+                    <Box variant='tokenTable' width='12rem' height='3rem'>
+                      <Account accountId={tokenSale?.toAddress} />
+                    </Box>
+                    <Box variant='tokenTable' width='12rem' height='3rem' />
+                    <Box variant='tokenTable' width='12rem' height='3rem'>
+                      <Text alignment={TextAlignment.Center}>
+                        { `Ξ${tokenSale.value / 1000000000000000000.0}`}
+                      </Text>
+                    </Box>
+                    <Box variant='tokenTable' width='12rem' height='3rem'>
+                      <Text alignment={TextAlignment.Center}>
+                        {dateToString(tokenSale.blockDate, 'HH:mm')}
+                      </Text>
+                    </Box>
+                  </Stack>
+                </ResponsiveHidingView>
+                <ResponsiveHidingView hiddenAbove={ScreenSize.Medium}>
+                  <Stack direction={Direction.Vertical} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
+                    <Stack direction={Direction.Horizontal} childAlignment={Alignment.Start} contentAlignment={Alignment.Start}>
+                      <Box variant='tokenTable' width='14rem' height='3rem'>
+                        <Text alignment={TextAlignment.Center}>
+                          { `Ξ${tokenSale.value / 1000000000000000000.0}`}
+                        </Text>
+                      </Box>
+                      <Box variant='tokenTable' width='14rem' height='3rem'>
+                        <Text alignment={TextAlignment.Center}>
+                          {dateToString(tokenSale.blockDate, 'HH:mm')}
+                        </Text>
+                      </Box>
+                    </Stack>
+                    <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center}>
+                      <Box variant='tokenTable' width='12rem' height='3rem'>
+                        <Account accountId={tokenSale?.fromAddress} />
+                      </Box>
+                      <Box variant='tokenTable' width='4rem' height='3rem'>
+                        <KibaIcon iconId='ion-arrow-forward' />
+                      </Box>
+                      <Box variant='tokenTable' width='12rem' height='3rem'>
```

Comment:
this looks crazy - you've put the variant on lots of indiviudal boxes?! why not put all of this in one box doesnt that make sense?